scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -262,7 +262,7 @@
         (reduce (fn [task-ents-by-user task-ent]
                   (let [user (util/task-ent->user task-ent)
                         f (if (= new-running-task-ent task-ent)
-                            (fnil conj (sorted-set-by (util/same-user-task-comparator-penalize-backfill)))
+                            (fnil conj (sorted-set-by (util/same-user-task-comparator)))
```

Comment:
Should the `nil` be handled by the function generating `user->sorted-running-task-ents` who initializes to a sorted set instead of nil?